assets/models/system/databricks-dolly-v2-3b/asset.yaml

```diff
@@ -0,0 +1,4 @@
+extra_config: model.yaml
```

Comment:
Could you add save to blobstorage run in the description?

Reply:
Added in the 
[https://ml.azure.com/experiments/id/9e347d5b-f23b-4e44-ae2f-63c30c22de62/runs/brave_animal_y24s7m7lxy?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)